staging/src/k8s.io/kms/encryption/managed_test.go

```diff
@@ -0,0 +1,200 @@
+package encryption_test
```

Comment:
```suggestion
package encryption
```

Reply:
Isn't it better to test "from the outside", without "inside knowledge"?